weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/OverflowMenu.tsx

```diff
@@ -159,33 +159,38 @@ export const ConfirmDeleteModal: FC<{
     Object.keys(projectGroups).forEach(projectKey => {
       const [entity, project] = projectKey.split('/');
       deletePromises.push(
-        callsDelete(entity, project, projectGroups[projectKey]).catch(() => {
-          const callNames = calls.map(callDisplayName);
-          setError(`Failed to delete call(s) ${callNames.join(', ')}`);
-        })
+        callsDelete(entity, project, projectGroups[projectKey])
       );
     });
-    Promise.all(deletePromises).then(() => {
-      userEvents.deleteClicked({
-        callIds: calls.map(
-          call => `${call.entity}/${call.project}/${call.callId}`
-        ),
-        numCalls: calls.length,
-        userId: userInfoLoaded?.id ?? '',
-        organizationName: orgName,
-        entityName: userInfoLoaded?.username ?? '',
+    Promise.all(deletePromises)
+      .then(() => {
+        userEvents.deleteClicked({
```

Comment:
Not sure how we are using this event but it seems a little off
- it gets logged on success, not click
- why is field called "entityName" if it is capturing the username?